trie/branch.go

```diff
@@ -154,29 +153,42 @@ func (b *branch) hash() hash.Hash32B {
 	return blake2b.Sum256(stream)
 }
 
+func (b *branch) toProto() *iproto.NodePb {
```

Comment:
I prefer call it `ToProto` as we have widely use throughout the code. Later on, we could extract interface from this.

Reply:
after discussion, this is an internal data struct, will keep as-is, can change to public if we need to expose interface from this